abr_control/interfaces/__init__.py

```diff
@@ -1,2 +1,6 @@
 from .vrep import VREP
-from .pygame import PyGame
+try:
```

Comment:
need to check examples / implications of not having this ... also do we need to do this anywhere else?

Reply:
I think it should just be in interface and examples

Reply:
oh i meant any other libraries too, was thinking maybe cython and the 3linkarm?